website/files/utils.py

```diff
@@ -16,12 +16,18 @@ def copy_files(src, target_node, parent=None, name=None):
     cloned.save()
 
     if src.is_file and src.versions.exists():
-        cloned.versions.add(*src.versions.all())
-        most_recent_fileversion = cloned.versions.select_related('region').order_by('-created').first()
-        if most_recent_fileversion.region != target_node.osfstorage_region:
-            most_recent_fileversion.region = target_node.osfstorage_region
-            most_recent_fileversion.save()
-
+        fileversions = src.versions.select_related('region').order_by('-created')
+        most_recent_fileversion = fileversions.first()
+        if most_recent_fileversion.region and (most_recent_fileversion.region != target_node.osfstorage_region):
```

Comment:
Decided to add a check to make sure the fileversion has a region set before performing the other check because of the way we create and use FileVersions in our tests.  It seems like mostly a FileVersion is created from the factory first and then added to a file afterwards manually. When they're created since they're not attached to a `BaseFileNode` yet, there's not a good default region to set them to to match their file's nodesettings region.

So in short I decided to leave the failing test alone, and instead check for a region on the fileversion (tho they should always be there for real) so that future tests didn't have to worry about region on FileVersions when they don't need to.